src/user/__snapshots__/user.spec.js.snap

```diff
@@ -6,7 +6,7 @@ exports[`user .create() failed : tries to create a user without password 1`] = `
 
 exports[`user .create() fails trying to create a user with an email already used 1`] = `[JsonRpcError: unknown error from the peer]`;
 
-exports[`user .create() successfully : creates a user with permission 1`] = `
+exports[`user .create() succeed : creates a user with permission 1`] = `
```

Comment:
My bad is succeeded. should we use a noun instead. describing success/failure? @julien-f